src/main/java/com/github/ygimenez/method/Pages.java

```diff
@@ -130,52 +130,44 @@ public static MessageHandler getHandler() {
 	 */
 	public static void paginate(@Nonnull Message msg, @Nonnull List<Page> pages) throws ErrorResponseException, InsufficientPermissionException {
 		if (!isActivated()) throw new InvalidStateException();
-		List<Page> pgs = Collections.unmodifiableList(pages);
+		List<Page> pageList = Collections.unmodifiableList(pages);
+		List<Emote> reactions = List.of(PREVIOUS, CANCEL, NEXT);
 		clearReactions(msg);
-
-		msg.addReaction(paginator.getEmotes().get(PREVIOUS)).submit();
-		msg.addReaction(paginator.getEmotes().get(CANCEL)).submit();
-		msg.addReaction(paginator.getEmotes().get(NEXT)).submit();
+		addReactions(msg, reactions);
 
 		handler.addEvent(msg, new ThrowingBiConsumer<>() {
-			private final int maxP = pgs.size() - 1;
-			private int p = 0;
-			private final Consumer<Void> success = s -> {
+			private final int lastPageIndex = pageList.size() - 1;
+			private int pageIndex = 0;
+			private final Consumer<Void> successConsumer = s -> {
```

Comment:
It's unnecessary to rename this one since it's self-explanatory (ie. a success action)